src/api/Versions.js

```diff
@@ -74,23 +122,61 @@ class Versions extends OffsetBasedAPI {
             size,
             version_number,
         };
+        const restoredFromId = getProp(file, 'restored_from.id');
+        const restoredVersion =
+            restoredFromId && entries.find((version: BoxItemVersion) => version.id === restoredFromId);
 
-        if (restored_from) {
-            const { id: restoredFromId } = restored_from;
-            const restoredVersion = versions.entries.find((version: BoxItemVersion) => version.id === restoredFromId);
-
-            if (restoredVersion) {
-                currentVersion.action = VERSION_RESTORE_ACTION;
-                currentVersion.version_restored = restoredVersion.version_number;
-            }
+        if (restoredVersion) {
+            currentVersion.action = VERSION_RESTORE_ACTION;
+            currentVersion.version_restored = restoredVersion.version_number;
         }
 
         return {
-            entries: [...versions.entries, currentVersion],
-            total_count: versions.total_count + 1,
+            entries: [...entries, currentVersion],
+            total_count: total_count + 1,
         };
     }
 
+    /**
+     * API for deleting a version of a file
+     *
+     * @param {string} fileId - a box file id
```

Comment:
Minor: should these follow http://usejsdoc.org/tags-param.html#parameters-with-properties?
i.e.
```
@param {object} config
@param {string} config.fileId
...
```